contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -292,11 +306,12 @@ contract ProtocolShareReserve is AccessControlledV8, ReentrancyGuardUpgradeable,
      */
     function _accrueAndReleaseFundsToPrime() internal {
         address[] memory markets = IPrime(prime).allMarkets();
-        for (uint256 i = 0; i < markets.length;) {
+        _ensureMaxLoops(markets.length);
```

Comment:
I think this check should be done in the Prime contract, when new markets are added, not here, because it would be too late